README.md

```diff
@@ -1,112 +1,137 @@
-    ```
+The python version is realized using the Python API for preCICE. Check the [preCICE wiki](https://github.com/precice/precice/wiki/1D-elastic-tube-using-the-Python-API) for more information on this example.
 
-2. After the script exits, you can view the output `.vtk` files in the `VTK` folder located in the root directory.
+### Building the C++ Solver
 
-3. To clean up the log files and vtk files created during a run, execute the `Allclean` script.
-    ```bash
-    $ ./Allclean
-    ```
-    
-**Optional:** Visualization and video output can be triggered via the options `--enable-plot` and `--write-video` of `FluidSolver.py`. If you want to use `Allrun`, you can set these options via environment variables:
+In order to use the C++ solver, you first need to build the scripts `FluidSolver` and `SolidSolver`. Each script needs to be built separately.
 
-* `$ ENABLE_PLOT=1 ./Allrun`: plots the simulation over time
-* `$ ENABLE_PLOT=1 WRITE_VIDEO=1 ./Allrun`: plots the simulation over time and creates a video.
+```
+cd fluid-cpp
+mkdir build && cd build
+cmake ..
+make all
+```
 
----
-## C++ version
+```
+cd solid-cpp
+mkdir build && cd build
+cmake .. 
+make all
+```
 
-Check [this preCICE wiki page](https://github.com/precice/precice/wiki/Example-for-FSI:-1D-elastic-tube) for a detailed description of this tutorial. For more information see [1]. Elastictube scenario taken from [2].
+Building can be skipped if you do not plan to use the C++ version.  
 
-### Requirements
+## Running the Simulation 
 
-- [preCICE](https://github.com/precice/precice/wiki/Get-preCICE)
+### C++
 
-- [LAPACK](http://performance.netlib.org/lapack/#_lapack_version_3_8_0_2). On Ubuntu-like Linux distributions, you can also install via executing:
-  ```bash
-  $ sudo apt-get install liblapack-dev
-  ```
+Open two separate terminals and start each participant by calling the respective run script. 
 
-### How to run
+Serial run:
 
-0. Clone this repository:
-    ```bash
-    $ git clone https://github.com/precice/elastictube1d.git
-    ```
+```
+./fluid-cpp/build/FluidSolver precice-config.xml N tau kappa
+```
+and
+```
+./solid-cpp/build/SolidSolver precice-config.xml N
+```
+ 
+Parallel run:
 
-1. Navigate into the `cxx` directory and build the Makefiles:
-   ```bash
-   $ cd cxx/ && cmake .
-   ```
-   **Note:** if `cmake` cannot find `libprecice.so`, please make sure that you are [linking to preCICE correctly](https://github.com/precice/precice/wiki/Linking-to-preCICE#linking-from-cmake).
+```
+mpiexec -np <nproc> ./fluid-cpp/build/FluidSolver precice-config.xml N tau kappa -parallel
+```
+and
+```
+mpiexec -np <nproc> ./solid-cpp/build/SolidSolver precice-config.xml N -parallel
+```
+A working known combination for the input parameters is N=100, tau = 0.01, kappa = 100. Other parameters like the simulation's end time, you can modify them in the precice-config.xml.
 
-2. Make the tutorial:
-   ```bash
-   $ make all
-   ```
+### python
```

Comment:
Let's use `Python` with uppercase `P` everywhere in the `README.md`, for consistency with the rest of the documentation.